challenges/selectionSort/selection-sort.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function selectionSort(intArr) {
+  let n = intArr.length;
+  for (let i = 0; i <= n-1; i++) {
+    let min = i;
+    for (let j = i + 1; j !== n; j++) {
+      if (intArr[j] < intArr[min]) min = j;
+    }
+    let temp = intArr[min];
+    intArr[min] = intArr[i];
+    intArr[i] = temp;
+  }
+}
+
+module.exports = { selectionSort };
```

Comment:
Hey Peter, this is select sort algorithm and not insert sort, please change this code or you can keep this and just add a new file that has insert sort following the pseudo code that you are given in the code challenge explanation